build/ci/jenkins_pipeline_build

```diff
@@ -48,4 +93,4 @@ pipeline {
         }
 
     }
-}
+}
```

Comment:
GitHub is telling us that a newline is missing here

Reply:
why do we need a new line between } and }?

Reply:
we don't. we need a new line between } and the EOF, like [every Posix text file](https://stackoverflow.com/a/729795).